policyengine_us/parameters/gov/states/nc/ncdhhs/scca/childcare_market_rates.yaml

```diff
@@ -0,0 +1,912 @@
+description: North Carolina set the following Subsidized Child Care Market Rates for Child Care Centers.
+metadata:
+  unit: currency-USD
```

Comment:
could you add a label?

Reply:
please make labels concise, can use abbreviations, see other parameters